README.md

```diff
@@ -195,6 +195,57 @@ buildctl build \
 
 `--local` exposes local source files from client to the builder. `context` and `dockerfile` are the names Dockerfile frontend looks for build context and Dockerfile location.
 
+##### Hint: `dockerfile.v0` frontend with `buildctl build --opt` option
+For `dockerfile.v0` frontend there are some parameters that can be used with `--opt`.
+The following parameters are the most frequently used.
+* target: set the target build stage to build.
+  * Example: `target=Stage1`
+* filename: name of the Dockerfile (Default is 'Dockerfile'),set to use a differently-named Dockerfile.
+  * Example: `filename=dockerfile.prod`,`filename=other.dockerfile`
+* dockerfilekey: dir path of the custom named context for Dockerfile. Useful when there is more than one build source.
```

Comment:
Just going by your example, it is the name of the context, not the directory path.

Reply:
yes.I will fixed this,it is name of the custom named context for a directory includes Dockerfile.